### DISCUSSION TITLE
How does the internal search function in GlobaLeaks work with multiple questionnaires?

### DISCUSSION THREAD

**Discussion by javierxul** [globaleaks/globaleaks-whistleblowing-software#4096] [General]
Hello,

I am using the GlobaLeaks platform and have several questions regarding the internal search functionality related to whistleblowing reports. In my current setup, I have created multiple questionnaires with various fields.

Which specific fields does the internal search function search within?
Does the search encompass all fields across all created questionnaires, or is it limited to certain predefined fields?
Is it possible to configure which specific fields should be searchable within each questionnaire?
I would appreciate any guidance or documentation you can provide on how the internal search works and how to adjust the searchable fields.

Thank you for your help.

**Comment by evilaliv3** [comment]
The current search is able to do a full search on the free text questions only.

**Reply by javierxul** [reply]
Thanks.
